node/websocket/websocket.go

```diff
@@ -488,6 +493,10 @@ func (c *connection) parseBlockResponse(response *jsonrpc.RawResponse) (*eth.Blo
 		return nil, errors.New(string(*response.Error))
 	}
 
+	if len(response.Result) == 0 || bytes.Equal(response.Result, json.RawMessage("null")) {
```

Comment:
Minor: "null" vs \`null\` in the other one.

https://github.com/INFURA/go-ethlibs/pull/6/files#diff-5384cc70ee9cf1d4bd3bafcb6bc3a8f6R548

(I've been thinking of making a jsonIsNull helper or something)